crates/bytecode/src/eof/header.rs

```diff
@@ -32,10 +32,13 @@ const KIND_TERMINAL: u8 = 0;
 const KIND_CODE_INFO: u8 = 1;
 const KIND_CODE: u8 = 2;
 const KIND_CONTAINER: u8 = 3;
-const KIND_DATA: u8 = 4;
+const KIND_DATA: u8 = 0xFF;
```

Comment:
Kind date moved to the end as data is always appended:
From doc https://notes.ethereum.org/dPK4iMNLTZ666Qn9zUNe1Q :
* Change kind_data from 0x04 to 0xff [#9502](https://github.com/ethereum/EIPs/pull/9502).